core/summary_api.php

```diff
@@ -649,86 +643,37 @@ function summary_print_by_category() {
 
 	$t_result = db_query( $t_query );
 
-	$t_last_category_name = -1;
-	$t_last_project = -1;
+	$t_cache = array();
+	$t_category_name = "";
+	$t_project = "";
+	$t_bugs_total_count = 0;
 	$t_bugs_open = 0;
 	$t_bugs_resolved = 0;
 	$t_bugs_closed = 0;
```

Comment:
The above 3 assignments are unnecessary, the variables' values are overwritten in the foreach loop